troubleshooting-input-type-file-not-working-on-android.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Input Type File Not Working on Android
+description: HTML input element of type file is not working in the Android web view.
+type: troubleshooting
+page_title: HTML File Input Not Responding in the Android Web View
+slug: troubleshooting-input-type-file-not-working-in-android
+position:
+tags: appbuilder, android
+teampulseid:
+ticketid:  1002377,  1020566, 1018666
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Android</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS version</td>
+    <td>4.4 KitKat and above</td>	
+  </tr>  
+</table>
+
+## Description
+Using `<input type="file">`
```

Comment:
Do you need the line breaker here? It should not affect the rendering of the paragraph but it looks weird in the source.